README.md

```diff
@@ -1 +1,12 @@
-# proposals
\ No newline at end of file
+# Stencil Community Proposals
+
+Use this repository to create proposals for new Stencil Community packages.
+
+## Usage
+
+Proposals are written in issues.
```

Comment:
Consider - adding a link to issue creation here since it's the first time we reference it in the doc

```suggestion
Proposals are written in [issues in this repository](https://github.com/stencil-community/proposals/issues).
```